debezium-server-eventhubs/src/main/java/io/debezium/server/eventhubs/EventHubsChangeConsumer.java

```diff
@@ -89,6 +92,10 @@ void connect() {
         maxBatchSize = config.getOptionalValue(PROP_MAX_BATCH_SIZE, Integer.class).orElse(0);
         configuredPartitionId = config.getOptionalValue(PROP_PARTITION_ID, String.class).orElse("");
         configuredPartitionKey = config.getOptionalValue(PROP_PARTITION_KEY, String.class).orElse("");
+        if (!configuredPartitionId.isEmpty() || !configuredPartitionKey.isEmpty())
+            dynamicPartitionRouting = "default";
+        else
+            dynamicPartitionRouting = config.getOptionalValue(PROP_DYNAMIC_PARTITION_ROUTING_KEY, String.class).orElse("default");
```

Comment:
Could you use enums instead of raw strings ("default", "key" and "partitionid")?